modules/timeutil/since.go

```diff
@@ -131,11 +147,21 @@ func timeSinceUnix(then, now time.Time, lang translation.Locale) template.HTML {
 }
 
 // TimeSince renders relative time HTML given a time.Time
-func TimeSince(then time.Time, lang translation.Locale) template.HTML {
+func TimeSince(ctx context.Context, then time.Time, lang translation.Locale) template.HTML {
+	// if user forces absolute timestamps, use the full time
+	val, err := preferenceHelper.GetSetting(ctx, preferenceHelper.SettingsForceAbsoluteTimestamps, "false")
```

Comment:
This `false` here is the default, right? I assume this usage isn't ideal because if the setting is queried in multiple places, the default would have to be declared at all call sites. Can we somehow move the default to a shared place, avoiding the issue?

Reply:
Was this resolved?

Reply:
I will try

Reply:
Done in https://github.com/go-gitea/gitea/pull/24342/commits/c98fcfbe635c85db3f25112a16cf683afb3e2e20

Reply:
Looking  good, thanks.